.github/workflows/linux-x64.yaml

```diff
@@ -9,6 +9,10 @@ jobs:
   build:
     runs-on: [self-hosted, linux, X64]
     steps:
+      - name: Install .NET SDK
+        uses: actions/setup-dotnet@v2
+        with:
+          dotnet-version: '6.0.x'
```

Comment:
I think this change is not needed: since the tests are Dockerized, your changes in `.docker/standalone.Dockerfile` will install .NET 6.0.x directly into the Docker image within which tests will be carried out. (No F* tests are run directly from this workflow outside of that Docker image.)